src/debugfs/stats.c

```diff
@@ -509,8 +509,13 @@ static int nrf_wifi_wlan_fmac_dbgfs_stats_show(struct seq_file *m, void *v)
 #endif /* DEBUG_MODE_SUPPORT */
 	int op_mode = RPU_OP_MODE_MAX;
 	struct rpu_op_stats *stats = NULL;
-
+#ifdef CONFIG_NRF700X_RADIO_TEST
+	unsigned int i = 0;
+#endif
 	rpu_ctx_lnx = (struct nrf_wifi_ctx_lnx *)m->private;
+#ifdef CONFIG_NRF700X_RADIO_TEST
+	op_mode = rpu_ctx_lnx->conf_params.op_mode,
+#endif
```

Comment:
In general we are following closing guards in comments

Reply:
This is no more needed. Separated radio test and STA mode functionalities as per below comment.

Reply:
I was referring to below but we can do that later.

`#endif /* CONFIG_NRF700X_RADIO_TEST */`